src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -134,13 +134,37 @@
 		</StackPanel>
 
 		<StackPanel
+			x:Name="GitActionsPanel"
 			Grid.Column="1"
 			VerticalAlignment="Center"
+			x:Load="{x:Bind DirectoryPropertiesViewModel.GitBranchDisplayName, Mode=OneWay, Converter={StaticResource NullToFalseConverter}}"
 			Orientation="Horizontal"
 			Spacing="8">
+
+			<!--  Pull Button  -->
+			<Button
+				x:Name="GitPullButton"
+				VerticalAlignment="Center"
+				Background="Transparent"
+				BorderThickness="0"
+				Command="{x:Bind Commands.GitPull}"
+				IsEnabled="{x:Bind Commands.GitPull.IsExecutable, Mode=OneWay}"
+				ToolTipService.ToolTip="{x:Bind Commands.GitPull.LabelWithHotKey, Mode=OneWay}">
+				<StackPanel Orientation="Horizontal" Spacing="4">
+					<FontIcon
+						HorizontalAlignment="Center"
+						FontSize="12"
+						Glyph="{x:Bind Commands.GitPull.Glyph.BaseGlyph}" />
+
+					<InfoBadge
```

Comment:
Can you switch this to regular text? This will allow us to display the number of commits available for pushing (in a future update).